src/strawchemy/schema/factories/base.py

```
@@ -278,34 +298,79 @@ def _input_wrapper(
+        aliases: Mapping[str, str] | None = None,
+        alias_generator: Callable[[str], str] | None = None,
+        name: str | None = None,
+        description: str | None = None,
+        directives: Sequence[object] | None = (),
+        override: bool = False,
+        purpose: Purpose = Purpose.WRITE,
+        scope: DTOScope | None = None,
+        base: type[Any] | None = None,
+        user_defined: bool = False,
+        no_cache: bool = False,
+        **kwargs: Any,
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)